### ISSUE TITLE
Videos not loading

### ISSUE THREAD

**Issue by thisisme345** [routayush1/animexninja#53]
Videos are not loading.

**Comment by geetesh07** [comment]
Do you know how to install it? The only thing that happens is the box is running for me nothing else

**Comment by animeawake** [comment]
Everything is working now its FIXED

**Comment by geetesh07** [comment]
Will you share your knowledge on how to take it live on hosting?

**Comment by geetesh07** [comment]
> Everything is working now its FIXED

No it is working, but the video starts to download as soon as u load the anime episode? Is it normaL?

**Comment by PaimonWheeWhee** [comment]
It's not fixed, Gogoanime just turned captcha off for their website. Problems will start showing again when they turn it back on

**Comment by animemandir** [comment]
Yeah, the episodes start to download when we click on them........

**Comment by geetesh07** [comment]
> Yeah, the episodes start to download when we click on them........
 Exactly, and i dont see the owner replying any soon either so ig we gotta find something

**Comment by animemandir** [comment]
hey @animeawake 
how exactly did you run the project, can you please show me steps, we can talk about on discord or insta.

**Comment by animemandir** [comment]
I think the problem is with gogoanime,
All the sites which fetch content from gogocdn 
even animy.ml which scrapes gogo has stopped due this problem, maybe the api url has changed or something.

**Comment by geetesh07** [comment]
> I think the problem is with gogoanime,
> All the sites which fetch content from gogocdn
> even animy.ml which scrapes gogo has stopped due this problem, maybe the api url has changed or something.

A lot of websites work from gogo anime and others are working fine.. the only thing author needs to do is add a player for all the videos.. like JW player or plyr.io and link it with the scraped urls... you can see the example in animekisa.cc or any other site

**Comment by ArunTeltia** [comment]
The video not working it is simply downloading the episode
